.github/workflows/build_and_test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     uses: ./.github/workflows/_build.yml
     with:
       ecr-docker-image-base: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/xla_base
-      gcr-docker-image: gcr.io/tpu-pytorch/xla_base:latest
```

Comment:
Thanks for upgrading this! This image seems to built on Feb 2023: gcr.io/tpu-pytorch/xla_base:latest